examples/browser/package.json

```diff
@@ -25,15 +25,16 @@
     "@theia/terminal": "^0.3.0",
     "@theia/typescript": "^0.3.0",
     "@theia/userstorage": "^0.3.0",
-    "@theia/workspace": "^0.3.0"
+    "@theia/workspace": "^0.3.0",
+    "@theia/keymaps": "^0.3.0"
   },
   "scripts": {
     "prepare": "yarn run clean && yarn build",
     "clean": "theia clean && rimraf errorShots",
     "build": "theia build",
     "watch": "yarn build --watch",
     "start": "theia start",
-    "start:debug": "yarn start --logLevel=debug",
+    "start:debug": "yarn start --loglevel=debug",
```

Comment:
it is `logLevel`, not in the lower case: https://github.com/theia-ide/theia/blob/6cfdece584c4615dc0b7b3c4a6b33faddcb91fab/packages/core/src/node/bunyan-logger-server.ts#L21

Reply:
I have no idea how I ended up modifying that (and I didn't notice it when doing `git add --patch`!